### PR TITLE
Fix for mapy.cz and landscan.ornl.gov

### PR DESCRIPTION
**Pull request by Tinh1000000**
Invert map tiles used on 2D maps
Fix map markers with improper appearances while planning directions (.icon polygon & CSS)

### CODE REVIEW

**Comment by Tinh1000000** [comment]
For landscan.ornl.gov:

Invert OSM tiles used for map
Adjust population density legend to use original key colours (manually specifying them individually for now, I do not immediately know of a way to specifically ignore changes to the legend)

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -13357,6 +13357,60 @@ INVERT
 
 ================================
 
+landscan.ornl.gov
```
Comment (Myshor) [inline src/config/dynamic-theme-fixes.config:13360]: As for maps.cz it seems it works nice but for this map is not.

Have you zoomed map to some city when default landscan is enabled?
Without your fix it's like this:
obraz

but after updates it is more eye-painfull:
obraz

Not sure if I would like to touch this map at all.

**Review by Myshor — CHANGES_REQUESTED** [review]
I reject for the part of landscan.ornl.gov

**Review by Myshor — APPROVED** [review]

**Comment by Myshor** [comment]
Now it looks darker... Merging. 😉

**Pull request merged**